leveldb-server/src/db.ts

```diff
@@ -72,8 +79,9 @@ export class Handler implements DBHandler {
   protected emitter = new EventEmitter();
   protected readonly writeLock = new Mutex();
   protected readonly db: LevelUp;
-  constructor(protected readonly dbPath: string) {
-    this.db = new LevelUpCtor(new LevelDown(dbPath));
+
+  constructor(protected readonly dbPath: string, errorCallback?: (err: Error) => any) {
```

Comment:
Note the lint failure highlights that types of err here is missing undefined (the levelup type is `(err: Error | undefined) => void`).

Reply:
Thanks!

Strange the compiler didn't catch it, I'm passing an `errorCallback` that cannot handle `undefined` inputs to a function that says it might pass `undefined`.  Stranger still, while playing around with it I'm pretty sure I saw `null` being passed.  We shall see.